outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -99,115 +91,136 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
 
         }).catch((error) => {
             console.log(error);
-            this.enriched = true;
             this.setState({isCollapsed: false, isLoading: false});
             this.context.showHttpErrorMessage(error);
         });
 
     }
 
-    private getCompanyLayout= () => {
-        const company = this.state.company as Company;
+    private enrichAndUpdate = () => {
+        if (!this.props.partner.email) {
+            let enrichmentInfo = new EnrichmentInfo(EnrichmentInfoType.EnrichContactWithNoEmail,
+                _t("This company has no email address and it couldn't be enriched."));
```

Comment:
Same here for translation + english

Reply:
Same thing, must be added in the forward port.